mne/viz/misc.py

```diff
@@ -107,19 +107,30 @@ def plot_cov(cov, info, exclude=[], colorbar=True, proj=False, show_svd=True,
                         'channels.')
 
     import matplotlib.pyplot as plt
+    from matplotlib.colors import Normalize
 
     fig_cov, axes = plt.subplots(1, len(idx_names), squeeze=False,
-                                 figsize=(2.5 * len(idx_names), 2.7))
+                                 figsize=(3.8 * len(idx_names), 3.7))
     for k, (idx, name, _, _) in enumerate(idx_names):
```

Comment:
You could also do `for ax, (idx, name, _, _, _) in zip(axes.flatten(), idx_names):`, and it would make the rest of the code a bit nicer, right?